README.md

```diff
@@ -30,52 +30,53 @@ _Data is initially pulled from Stanford's system, but the user can update certai
 
 | Column Name | Type | Meaning | Notes |
 | - | - | - | - |
-| `SUNet` | string (primary key) | SUNet id | never shown to users, only used internally |
-| `first_name` * | string | first name |  |
-| `last_name` * | string| last name |  |
-| `email` * | string (email address) | user's email address | assumed initially to be SUNet@stanford.edu, but user can update |
```

Comment:
We probs don't want to store the above 4 fields for security reasons. We'd have to encrypt at least the email and sunet (and probs first and last name tbh). We should just rely on whatever stanford auth sends us if possible.

Reply:
I figured it would be okay because all of that info is already public (https://stanfordwho.stanford.edu/, for example), especially if we limit the email to Stanford emails. Not sure how it works in industry tho, so if you say that's best I believe you!

Reply:
it looks like you have to log into that site to get all the info outside of someone's name. I'm not particularly worried about stanford students getting this info, more-so about nefarious non-stanford users. If for some reason Stanford auth doesn't give us the first / last name, that should be ok to store, but I imagine that this will be given to us.